torch/distributed/_tensor/examples/convnext_example.py

```diff
@@ -0,0 +1,260 @@
+"""
+The following example demonstrates how to train a ConvNeXt model
+with intermediate activations sharded across mutliple GPUs via DTensor
+"""
+import os
+import time
+import warnings
+
+import torch
+import torch.distributed as dist
+import torch.multiprocessing as mp
+import torch.nn as nn
+from torch.distributed._tensor import (
+    DeviceMesh,
+    distribute_module,
+    distribute_tensor,
+    Replicate,
+    Shard,
+)
+
+warnings.filterwarnings("ignore", category=UserWarning)
```

Comment:
why we need to filter out warning here?

Reply:
Thanks for reminding. I have removed it.